src/group_elements.jl

```diff
@@ -1,223 +1,225 @@
-## `GroupElement` interface
+################################################################################
+#
+#   group_elements.jl : Interface for group elements
+#
+################################################################################
 
-### Obligatory methods for `GroupElement`
+# Creator : Marek Kaluba
 
-Base.parent(g::GroupElement) =
-    throw(InterfaceNotImplemented(:Group, "Base.parent(::$(typeof(g)))"))
+################################################################################
+# Obligatory methods
+################################################################################
 
 @doc Markdown.doc"""
-    parent_type(element_type)
-Given the type of an element return the type of its parent.
-"""
-AbstractAlgebra.parent_type(GEl::Type{<:GroupElement}) = throw(
-    InterfaceNotImplemented(:Group, "GroupsCore.parent_type(::Type{$GEl})"),
-)
-
-@doc Markdown.doc"""
-    ==(g::GEl, h::GEl) where {GEl<:GroupElement}
-Return the mathematical equality of group elements.
+    ==(g::G, h::G) where {G <: GroupElement}
 
-This function may not return due to e.g. unsolvable word problem in groups.
+Return true if $g = h$ and it is computable, else return error.
 """
-Base.:(==)(g::GEl, h::GEl) where {GEl<:GroupElement} = throw(
-    InterfaceNotImplemented(:Group, "Base.:(==)(::$GEl, ::$GEl)"),
+Base.:(==)(g::G, h::G) where {G <: GroupElement} = throw(
+    InterfaceNotImplemented(:Group, "Base.:(==)(::$G, ::$G)"),
 )
 
 @doc Markdown.doc"""
     isfiniteorder(g::GroupElement)
-Return `true` if `g` has finite order (without computing it).
+
+Return true if $g$ is of finite order without computing it.
```

Comment:
In the README, it says `(possibly without computing it)`, while here it says `(without computing it)`. Which one is right?